docs/source/reference/cooperative_groups.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+
+Cooperative groups
```

Comment:
How about adding them in `kernel.rst`?

Reply:
Yes it makes sense too because the returned groups are public API. Will change.

Reply:
Done in [ce85e2d](https://github.com/cupy/cupy/pull/6620/commits/ce85e2d7164471bac43d60f0f5b15b2151a3aa0e).

Reply:
I noticed that the class autosummary template would expose the parent class's methods (in this case, `declvar` and `assign`, see [here](https://cupy--6620.org.readthedocs.build/en/6620/reference/generated/cupyx.jit.cg._ThreadBlockGroup.html#cupyx.jit.cg._ThreadBlockGroup)), while Sphinx's `autoclass` by default turns it off. I feel it's a bit distracting...

Reply:
dba542457e may be better?

Reply:
Thanks!

Reply:
@asi1024 sorry, on a second thought `autoclass` doesn't look nice either... https://docs.cupy.dev/en/latest/reference/kernel.html#cooperative-groups
It expands all members on the otherwise index-only page. Time to reconsider making an orphan page for redirection?